sklearn/model_selection/tests/test_search.py

```diff
@@ -1885,3 +1887,27 @@ def _fit_param_callable():
         'scalar_param': 42,
     }
     model.fit(X_train, y_train, **fit_params)
+
+
+@pytest.mark.parametrize(
+    "Estimator,params,estimator_type",
+    [(DecisionTreeClassifier, {"max_depth": [5, 10]}, "classifier"),
+     (DecisionTreeRegressor, {"max_depth": [5, 10]}, "regressor"),
+     (KMeans, {"n_clusters": [5, 10]}, "clusterer"),
+     (BayesianGaussianMixture, {"n_components": [5, 10]}, "DensityEstimator"),
+     (IsolationForest, {"max_depth": [5, 10]}, "outlier_detector")]
```

Comment:
To make these faster, we can do use `[2, 3]`?

Reply:
I will change these values. Nevertheless, since the model is not fitted, is there any computational improvement?

Reply:
Resolving conversation because the hyper(parameters) are not considered in `parametrize` anymore.